at_client/lib/src/listener/sync_progress_listener.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:at_client/src/service/sync/sync_status.dart';
+
+abstract class SyncProgressListener {
+  void onSync(SyncProgress syncProgress);
```

Comment:
Please rename SyncProgressListener's `onSync` method to `onSyncProgressEvent`

Reply:
done Gary.